utils/upload_utils.py

```diff
@@ -40,9 +41,13 @@ def post_texts_with_geo(
         )
 
         post_inputs_response = stub.PostInputs(post_inputs_request)
```

Comment:
Any reason on why`PostInputs` is still present. You are uploading inputs using SDK below
```python
input_obj = Inputs(user_id=userDataObject.user_id, app_id = userDataObject.app_id)
post_inp=input_obj.upload_inputs(inputs)
```

Reply:
I think we can refactor forming input_proto from [get_text_input](https://github.com/Clarifai/clarifai-python/blob/2a662e77fa8d3f6880f28c8792b0bee4fb628cf1/clarifai/client/input.py#L259)
```python
inp = resources_pb2.Input(
                data=resources_pb2.Data(
                    text=resources_pb2.Text(raw=text),
                    geo=resources_pb2.Geo(
                        geo_point=resources_pb2.GeoPoint(
                            longitude=geo_points_batch[idx]["lon"],
                            latitude=geo_points_batch[idx]["lat"],
                        )
                    ),
                )
            )
```